.gitignore

```diff
@@ -1,9 +1,8 @@
 .DS_Store
-errors/**
```

Comment:
We need the `errors` folder, because the tests will generate it and I don't want them committed to the repo.